WordPress/src/test/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModelTest.kt

```diff
@@ -283,6 +320,24 @@ class SiteCreationDomainsViewModelTest {
         assertThat(uiState.contentState.items.size, Is(numberOfItems))
     }
 
+    /**
+     * Helper function to verify a [DomainsUiState] with [DomainsUiContentState.VisibleItems] content state & [DomainsModelUnavailabilityUiState] Ui State.
+     */
+    private fun verifyVisibleItemsContentUiAndDomainValidityState(
+        uiState: DomainsUiState,
+        showClearButton: Boolean = false,
+        numberOfItems: Int = MULTI_RESULT_DOMAIN_FETCH_RESULT_SIZE
+    ) {
+        assertThat(uiState.searchInputUiState.showProgress, Is(false))
+        assertThat(uiState.searchInputUiState.showClearButton, Is(showClearButton))
+        assertThat(uiState.contentState, instanceOf(DomainsUiContentState.VisibleItems::class.java))
+        assertThat(uiState.contentState.items.size, Is(numberOfItems.inc()))
```

Comment:
For this test, I simply increment the total number of items since when the `Unavailability` state gets added then an extra state item exists. Should this be more explicit or it's fine?

Reply:
Great 👍

Reply:
Great 👍